userspace/libsinsp/sinsp_inet.h

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 #pragma once
 
 #if defined(_WIN32)
+#define NOMINMAX
```

Comment:
I can guess what this does but how about a comment?

Reply:
It prevents including <windows.h> as an unwanted side-effect of including Ws2tcpip.h. I added a comment.